### PR TITLE
upgrade p2pserver

### PR DESCRIPTION
**Pull request by dylenfu**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -3,26 +3,16 @@ module map
...
-	github.com/gosuri/uilive v0.0.3 // indirect
-	github.com/gosuri/uiprogress v0.0.1 // indirect
-	github.com/hashicorp/golang-lru v0.5.1
-	github.com/howeyc/gopass v0.0.0-20170109162249-bf9dde6d0d2c // indirect
+	github.com/hashicorp/golang-lru v0.5.3
 	github.com/imroc/req v0.2.3
-	github.com/itchyny/base58-go v0.0.5 // indirect
-	github.com/ontio/ontology v1.6.2
-	github.com/ontio/ontology-crypto v1.0.4 // indirect
+	github.com/ontio/ontology v0.0.0-20200624063426-fcbf82cb6203
```
Comment (Honglei-Cong) [inline go.mod]: anchor to v2.0.0 ?

File: main.go [hunk 2]
```diff
@@ -1,22 +1,24 @@
 package main
 
 import (
+	"os"
+	"os/signal"
+	"runtime"
+	"syscall"
+
```
Comment (Honglei-Cong) [inline main.go:11]: remove blank lines

File: go.mod (hunk 1 above)
Comment (dylenfu) [reply go.mod]: ontology version above 2.0, marjor version here must be compatible or v0.0.0-20200624063426-fcbf82cb6203

File: main.go (hunk 2 above)
Comment (dylenfu) [reply main.go:11]: all fixed

**Pull request merged**